hack/integration-test/setup.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${BUILD_DEFS_REPO:?Missing repository path of build-definitions}"
+: "${QUAY_NAMESPACE:?Missing QUAY_NAMESPACE. build-and-push.sh uses it to push task bundles}"
+
+export QUAY_NAMESPACE
+
+declare -r DEFAULT_RECIPES="
+clone 0.2 -
+clone 0.2 M
+clone 0.2 -
+clone 0.2 M
+clone 0.2 -
+lint 0.2 -
+lint 0.2 M
+lint 0.2 M
+lint 0.2 M
+lint 0.2 -
```

Comment:
I noticed the setup takes a long time (more than 5 minutes on my machine). Probably because we run build-and-push.sh in a loop. Maybe we could at least reduce the number of tasks in the default recipe?